### PR TITLE
Fix a key error in enarxbot-pr-assign

### PR DESCRIPTION
**Pull request by mbestavros**
This should also prevent attempting to assign non-valid users (like bots) to PRs.

Signed-off-by: Mark Bestavros <mbestavr@redhat.com>

### CODE REVIEW

File: enarxbot-pr-assign [hunk 1]
```diff
@@ -85,10 +85,12 @@ def iterusers(nested_dictionary):
 
     for key, value in nested_dictionary.items():
         if key in ["author", "who"]:
-            yield value
+            if value:
```
Comment (npmccallum) [inline enarxbot-pr-assign:88]: I suspect you probably want `if value is not None:` in both cases.

**Review by npmccallum — CHANGES_REQUESTED** [review]

File: enarxbot-pr-assign (hunk 1 above)
Comment (mbestavros) [reply enarxbot-pr-assign:88]: @npmccallum We _do_ get a value in both cases -- not `None`. It's just an empty dictionary.

Comment (mbestavros) [reply enarxbot-pr-assign:88]: @npmccallum Here's an example printout of `value` from a run locally:

```
{'login': 'ambaxter', 'id': 'MDQ6VXNlcjE1MDA2Nzgw'}
{'login': 'npmccallum', 'id': 'MDQ6VXNlcjI4ODMwNA=='}
{'login': 'haraldh', 'id': 'MDQ6VXNlcjQyNzgyNQ=='}
{}
{'login': 'ueno', 'id': 'MDQ6VXNlcjQ3NTQ5'}

```

The empty dictionary is what's causing the problem.

Comment (npmccallum) [reply enarxbot-pr-assign:88]: Then check for `len(value) == 0`.

Comment (mbestavros) [reply enarxbot-pr-assign]: Fair enough. Amended.

**Review by npmccallum — APPROVED** [review]

**Pull request merged**